contrib/dist/linux/buildrpm.sh

```diff
@@ -45,12 +88,69 @@ build_multiple=${build_multiple:-"no"}
 #########################################################################
 
 #
+# save original parameters
+#
+orig_param="$@"
+
+#
+# usage information
+#
+usage="Usage: $0 [-b][-o][-m][-d][-u][-h] [-n <name>][-f <lf_location>][-t <tm_location>] <tarball>
+
+	-b                do NOT build SRPM, only all-in-one binary RPM (required for other flags to work)
+	-n <name>         name of the resulting RPM package (openmpi)
+	-o                install in /opt/<name>/<version>
+	-m                create modulefiles
+	-d                build debuginfo RPM
+	-u                include usNIC support
+	-f <lf_location>  include LibFabric support from <lf_location>
+	-t <tm_location>  include Torque support from <tm_location>
+	-h                print this message and exit
+	-s                try to unpack openmpi.spec file from <tarball>
+
+	<tarball>         path to OpenMPI source tarball
+  "
+
+#
+# parse args
+#
+while getopts bn:omduf:t:sh flag; do
+    case "$flag" in
+      b) build_srpm="no"
+         build_single="yes"
+         ;;
+      n) rpmbuild_options="$rpmbuild_options --define '_name $OPTARG'"
+         ;;
+      o) rpmbuild_options="$rpmbuild_options --define 'install_in_opt 1'"
+	 configure_options="$configure_options --enable-mpirun-prefix-by-default --enable-orterun-prefix-by-default"
```

Comment:
These two options are synonyms of each other -- you probably only need to list one of them.